### ISSUE TITLE
cannot join E6 server for longer than 60 seconds

### ISSUE THREAD

**Issue by schpinkeldinkel** [NillerMedDild/Enigmatica6#551]
**Modpack version**
0.3.0, client & server

**Issue**
friend can join, but gets booted after a short unspecified period of time

**Logs**
client:
https://paste.ubuntu.com/p/GMthVJMrMh/ - friends crash log, since have changed java to x64 1.8.0_241. occurs with and without optifine.

server excerpt:
https://paste.ubuntu.com/p/v8z3WRcf9r/ - In the server log, whenever the player tries to join, it will spit out lines about 'fetching packet for removed entity..'. At the said coordinates there is nothing but regular stone and dirt, I interacted with the world at this spot but it persists.

server latest.log:
https://paste.ubuntu.com/p/bRkJ73Z23T/
server debug.log:
https://p.teknik.io/VlkxJ
...

Me & another player have played without issue for ~3 days. Another player tries to join us, and can, but only for around a minute before the player gets booted for different reasons every time. All timeout reasons are similarly related to bad packets, for example:
corruptedframeexception: length wider than 21-bit
Internal Exception: io.netty.handler.codec.decoderexception: java.util.zip.DataFormatException: incorrect header check

This occurs when my friend has firewall completely disabled. This occurs when the draw distance is set very low (2) in server properties. cheers a lot

**Comment by NillerMedDild** [comment]
Thanks for the report :)

I'm not seeing any attempt at joining the server in the client log, could you verify that it's the correct log please?

**Comment by schpinkeldinkel** [comment]
no, thanks for the hasty support!!! :}

sir, his debug.log and latest.log are around 0.2gb (near 2 million lines each)
it is difficult to paste these logs! here is debug.log: https://p.teknik.io/NfIYf

**Comment by NillerMedDild** [comment]
I can't even open those links :/ could you perhaps have him try again, and upload that log?

**Comment by schpinkeldinkel** [comment]
the links still work for me it just takes a minute to load, I don't know another paste site which will allow such sized pastes. I have the files in dropbox but understand that this isn't something you will like to use:
https://www.dropbox.com/s/u0us3p27or76evk/debug.log?dl=0
https://www.dropbox.com/s/dtw5jqrpmxcxszi/latest.log?dl=0

I will make him try again to get smaller logs when I can, and post them here

**Comment by schpinkeldinkel** [comment]
Sir, your persistent support is commendable. I have solved this issue, this issue was not caused by your pack. This issue wasn't caused by firewall either. The issue was caused by the router/router settings. In changing to wireless connection, or changing ethernet port used on the router the issue was resolved. Worth noting in CMD: ipconfig /flushdns did not resolve the issue. Perhaps somebody will be helped by this information. I hope you continue with your great work here, thankyou for your work
